specification/registry/xr.xml

```diff
@@ -3559,6 +3581,15 @@ maintained in the default branch of the Khronos OpenXR GitHub project.
             <param>const <type>XrMarkerSpaceCreateInfoVARJO</type>* <name>createInfo</name></param>
             <param><type>XrSpace</type>* <name>space</name></param>
         </command>
+      
+      
+        <!-- commands for XR_ALMALENCE_digital_lens_control -->
+        <command successcodes="XR_SUCCESS,XR_SESSION_LOSS_PENDING" errorcodes="XR_ERROR_FUNCTION_UNSUPPORTED,XR_ERROR_HANDLE_INVALID,XR_ERROR_INSTANCE_LOST,XR_ERROR_SESSION_LOST">
+            <proto><type>XrResult</type> <name>xrSetDigitalLensStateALMALENCE</name></proto>
+            <param><type>XrSession</type> <name>session</name></param>
+            <param>const <type>XrDigitalLensControlALMALENCE</type>* <name>digitalLensState</name></param>
+        </command>  
```

Comment:
So in terms of naming convention, "State" is stuff that comes from the runtime to the application. "Info" (especially "CreateInfo") is one name for things that go from the app to the runtime, but I might suggest calling it maybe "xrSetDigitalLensControl" and "digitalLensControl" for the function name and parameter name respectively.

Reply:
Ok, will try to find more suitable names that do not interfere with standard naming convention.

Reply:
Names corrected